msteams-platform/apps-in-teams-meetings/teams-live-share-faq.md

```diff
@@ -54,13 +54,22 @@ No. Teams currently doesn't support encrypted media for tab applications.
 
 </details>
 
+<details>
+<summary><b>How many people can attend a Live Share session?</b></summary>
+
+Currently, 100 people is the maximum supported number of Live Share attendees.
```

Comment:
Hi Corina,

Can we rewrite this as "Currently, Live Share supports a maximum of 100 attendees per session."